### DISCUSSION TITLE
Importing presets from ffmpeg command line?

### DISCUSSION THREAD

**Discussion by philstopford** [CineEncoder/cine-encoder#41] [Ideas]
I wondered whether it might be possible for cine-encoder to be able to take a commandline from ffmpeg and generate a corresponding preset for use in the tool. This would make it easier to migrate presets from other tools (e.g. myFFmpeg, Handbrake, etc.) where those tools can show the ffmpeg syntax in-full.

**Comment by CineEncoder** [comment]
Good afternoon!
Theoretically, it would be possible to do this, but there are several problems related to the fact that in addition to ffmpeg, mkvpropedit is also used in the program, it will turn out a very confusing logic of the program. Now I'm working on simplifying the logic.
